python/kotekan/_version.py

```diff
@@ -524,7 +524,7 @@ def get_versions():
         # versionfile_source is the relative path from the top of the source
         # tree (where the .git directory might live) to this file. Invert
         # this to find the root from __file__.
-        for i in cfg.versionfile_source.split("/"):
+        for _ in cfg.versionfile_source.split("/"):
```

Comment:
I'd be tempted to ignore this one. As this file is produced by versioneer it'll make it annoying to see what's going on if we ever update versioneer and re-gen this file.

Reply:
I think we blackened that file btw :laughing: